### PR TITLE
Fix typo and use latest os

### PR DESCRIPTION
**Pull request by mayou36**
Shouldn't we use the "latest" of the OS instead?

and a small typo fix

### CODE REVIEW

File: pages/developers/gha_wheels.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ env:
   CIBW_BUILD_VERBOSITY: 1
   # Optional: support Universal2 for Apple Silicon with these two lines:
   CIBW_ARCHS_MACOS: auto universal2
-  CIBW_TEST_SKIP: "*univeral2:arm64"
+  CIBW_TEST_SKIP: "*universal2:arm64"
```
Comment (eduardo-rodrigues) [inline pages/developers/gha_wheels.md:56]: Good catch!

File: pages/developers/gha_wheels.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ The core of the work is down here:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, windows-2019, macos-10.15]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```
Comment (eduardo-rodrigues) [inline pages/developers/gha_wheels.md:106]: @henryiii has made the comment that this exposes us every now and then to issues with fresh versions - has happened. Dunno, really.

Comment (mayou36) [reply pages/developers/gha_wheels.md]: fair enough, makes sense and I am sure @henryiii knows that well. changed that back

**Comment by mayou36** [comment]
Fixed another typo I forgot to, other than that fine for me

**Review by eduardo-rodrigues — APPROVED** [review]
Thanks!

**Comment by henryiii** [comment]
It's better to keep the versions pinned for wheel building; you don't want an OS update to break things by surprise after you release (macOS 11 does not support Python 3.5, for example, so that wheel would vanish; Windows MSVC 2019 requires an extra configuration or it will add an extra dependency that not all Python copies have, etc). We can go with ubuntu-20.04 if we don't have that yet (it's actually a hair better).

**Comment by henryiii** [comment]
Thanks!

**Pull request merged**